oa-enterprise/lib/omniauth/strategies/cas.rb

```diff
@@ -8,9 +8,30 @@ class CAS
       autoload :Configuration, 'omniauth/strategies/cas/configuration'
       autoload :ServiceTicketValidator, 'omniauth/strategies/cas/service_ticket_validator'
 
+      attr_accessor :raw_info
+      alias_method :user_info, :raw_info
+
+      @uid_key = 'user'
+
+      def userHash
```

Comment:
add fields defined in the omniauth user_info hash. support multiple formats that it can be shared in